### DISCUSSION TITLE
Puzzled about bpm setting (music numpty)

### DISCUSSION THREAD

**Discussion by oldfogie** [hydrogen-music/hydrogen#1705] [Hydrogen on Windows]
Our band master has marked the time for our jigs as dotted crotchet = 120
its in 3/4 time so I set 180 bpm and it works.

However when I set the bpm for reels (4/4) to 120 bpm a metronome its telling me its 60bpm. 

Have I got something set wrong?

**Comment by theGreatWhiteShark** [comment]
I'm afraid I can't follow.

First of all Hydrogen does not really understand the concept of time measures and the 3/4 and 4/4 you can set at the top of the pattern editor is merely changing the size of the pattern and is not affecting the metronome at all.

So, you manually set bpm to 120 and widget is displaying and using 60 instead? Could you provide a step-by-step guide to reproduce this behavior?

**Comment by johnerrington** [comment]
It transpires that the fault was in the music notation: Some of the music was in common time, and some in 2/4 or 4/8
